compiler/rustc_ast/src/ast.rs

```diff
@@ -2896,14 +2913,11 @@ pub enum ItemKind {
     ///
     /// E.g., `use foo;`, `use foo::bar;` or `use foo::bar as FooBar;`.
     Use(UseTree),
-    /// A static item (`static`).
-    ///
-    /// E.g., `static FOO: i32 = 42;` or `static FOO: &'static str = "bar";`.
-    Static(P<Ty>, Mutability, Option<P<Expr>>),
+    Static(Box<StaticItem>),
```

Comment:
While `Box<T>` is nicer to match on thanks to box patterns, I don't see it used anywhere here before as everything is `P<T>`, so it would probably make sense to use `P<T>` as well (or question ourselves why `P<T>` is used in the first place)

Reply:
Almost all other enum variants use `Box`, which is why I picked `Box` instead of `P`.

Reply:
Oh, they do, mobile "find in page" mislead me, I forgot that GitHub just completely doesn't work for that. Seems fine then. I don't really understand the point behind `P` anyways.